### PR TITLE
doc: introduction: distinguishing NCS features

### PR DESCRIPTION
**Pull request by greg-fer**
Edited the introduction page to include distinguishing features similar to the section in Zephyr docs.
NCSDK-22322.

---- 

- [x] Approval from @wbober

### CODE REVIEW

File: doc/nrf/introduction.rst [hunk 1]
```diff
@@ -9,9 +9,24 @@ Introduction
...
+   The |NCS| offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
+
+**Pre-certified libraries**
+   The |NCS| is optimized for pre-certified libraries, including SoftDevice, Matter, and Thread.
+
+**Robust connectivity support**
+   The |NCS| supports a wide range of connectivity technologies, such as Ant, Bluetooth® Low Energy, Bluetooth mesh, LTE-M/NB-IoT/GPS, Matter, Sidewalk, TCP/IP, Thread, and Wi-Fi®.
+
+**Third-party integrations**
+   The |NCS| allows for using third-party and Nordic products within the SDK, such as AWS, nRFCloud, memfault, and more.
```
Comment (wbober) [inline doc/nrf/introduction.rst:32]:
```suggestion
   The |NCS| provides integrations with third-party and Nordic products within the SDK, such as AWS, nRFCloud, memfault, and more.
```

File: doc/nrf/introduction.rst [hunk 2]
```diff
@@ -9,9 +9,24 @@ Introduction
...
+   The |NCS| integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications and samples.
+
+**Extensible framework**
+   The |NCS| offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
+
+**Pre-certified libraries**
+   The |NCS| is optimized for pre-certified libraries, including SoftDevice, Matter, and Thread.
+
+**Robust connectivity support**
+   The |NCS| supports a wide range of connectivity technologies, such as Ant, Bluetooth® Low Energy, Bluetooth mesh, LTE-M/NB-IoT/GPS, Matter, Sidewalk, TCP/IP, Thread, and Wi-Fi®.
```
Comment (wbober) [inline doc/nrf/introduction.rst:29]: "In addition to connectivity technologies provided by Zephyr [list] it supports [list]".

File: doc/nrf/introduction.rst [hunk 3]
```diff
@@ -9,9 +9,24 @@ Introduction
...
+   The |NCS| includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
+
+**Based on Zephyr**
+   The |NCS| integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications and samples.
+
+**Extensible framework**
+   The |NCS| offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
+
+**Pre-certified libraries**
+   The |NCS| is optimized for pre-certified libraries, including SoftDevice, Matter, and Thread.
```
Comment (wbober) [inline doc/nrf/introduction.rst:26]: Provided pre-certified and optimized libraries.

File: doc/nrf/introduction.rst [hunk 4]
```diff
@@ -9,9 +9,24 @@ Introduction
...
 
-The |NCS| supports :ref:`Microsoft Windows, Linux, and macOS <requirements>` for development.
+**Advanced middleware and security**
+   The |NCS| includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
+
+**Based on Zephyr**
+   The |NCS| integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications and samples.
+
+**Extensible framework**
+   The |NCS| offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
```
Comment (wbober) [inline doc/nrf/introduction.rst:23]: I'd remove this.

File: doc/nrf/introduction.rst [hunk 5]
```diff
@@ -9,9 +9,24 @@ Introduction
...
-It integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications, samples, and protocol stacks such as Bluetooth® Low Energy, Bluetooth mesh, Matter, Thread/Zigbee, Wi-Fi®, and LTE-M/NB-IoT/GPS, TCP/IP.
-It also includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
+The |NCS| has the following distinguishing features:
 
-The |NCS| supports :ref:`Microsoft Windows, Linux, and macOS <requirements>` for development.
+**Advanced middleware and security**
+   The |NCS| includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
+
+**Based on Zephyr**
+   The |NCS| integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications and samples.
```
Comment (wbober) [inline doc/nrf/introduction.rst:20]: Put as first.

File: doc/nrf/introduction.rst [hunk 6]
```diff
@@ -9,9 +9,24 @@ Introduction
...
-It offers an extensible framework for building size-optimized software for memory-constrained devices as well as powerful and complex software for more advanced devices and applications.
+It supports :ref:`Microsoft Windows, Linux, and macOS <requirements>` for development.
 
-It integrates the Zephyr™ real-time operating system (RTOS) and a wide range of complete applications, samples, and protocol stacks such as Bluetooth® Low Energy, Bluetooth mesh, Matter, Thread/Zigbee, Wi-Fi®, and LTE-M/NB-IoT/GPS, TCP/IP.
-It also includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
+The |NCS| has the following distinguishing features:
 
-The |NCS| supports :ref:`Microsoft Windows, Linux, and macOS <requirements>` for development.
+**Advanced middleware and security**
+   The |NCS| includes middleware such as CoAP, MQTT, LwM2M, various libraries, hardware drivers, Trusted Firmware-M for security, and a secure bootloader (MCUboot).
```
Comment (wbober) [inline doc/nrf/introduction.rst:17]: In addition to middleware provided by Zephyr, adds extra libraries such as [list only these that are not available upstream].

Comment (greg-fer) [reply doc/nrf/introduction.rst]: Added some, but would appreciate a more thorough look at this.

**Review by DeidreCasey — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Comment by greg-fer** [comment]
@wbober , I will wait for merge request for your sign-off on this one.

**Review by wbober — APPROVED** [review]

**Review by b-gent — APPROVED** [review]

**Pull request merged**